.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- Reference the source of this Pull Request. -->
+<!-- Remove any which are not applicable. -->
+**Issue**: #
+
+**Ticket**:
+
+**Slack thread**:
+
+---
+
+### Description
+<!-- Describe the changes made in this Pull Request and the reason for these changes. -->
+
+### Steps to test: 
+<!-- Describe the steps to replicate the issue and confirm the fix -->
+<!-- Try to include as many details as possible. -->
+1.
+1.
+1.
+
+### Documentation
+<!-- Will this change require new documentation or changes to existing documentation? -->
+<!-- A good way to answer it is to ask: will more than one customer ever need to know about this? -->
+- [ ] This PR needs documentation (has the "status:needs-docs" label).
```

Comment:
In this repo the label is `Documentation`

Reply:
Fixed :)